pkg/controller/build/buildrequest/buildrequest.go

```diff
@@ -525,7 +518,7 @@ func (br buildRequestImpl) toBuildahPod() *corev1.Pod {
 					// us to avoid parsing log files.
 					Name:            "wait-for-done",
 					Command:         append(command, waitScript),
-					Image:           br.opts.getBaseOSImagePullspec(),
+					Image:           br.opts.getBaseOSImagePullspec(), // QOCL: What do we replace this with?
```

Comment:
I'm guessing this would be our MC's base image?

Reply:
Correct. It's the value from `b.OSImageURLConfig.BaseOSContainerImage`.

Reply:
ack, will update!